apps/mobile/mobile-app/components/impact/impact-section.tsx

```
@@ -0,0 +1,85 @@
+			</View>
+			<Text className="text-typography-gray text-center text-base mb-4">
+				KindFi is more than a platform - it's a new way to fund the real change.
+				Whether you're a cause creator or a supporter, every action you take
+				earns trust, builds your on-chain reputation, and strenghtens a
+				transprarent, community-led ecosystem
+			</Text>
+
+			{impact_data.map((items, i) => {
+				return (
+					<ImpactCard
+						key={i}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)